flexmeasures/cli/data_add.py

```diff
@@ -369,6 +369,13 @@ def add_asset_type(**args):
     type=int,
     help="Asset type to assign to this asset",
 )
+@click.option(
+    "--parent-asset",
+    "parent_asset_id",
+    required=False,
+    type=int,
```

Comment:
We should have a Marshmallow field for this, something like `AssetIdField`.

Reply:
Good point, that way the command will break before and the error will be that the parent asset is missing.